src/nvim/plines.c

```diff
@@ -296,8 +296,16 @@ void init_chartabsize_arg(chartabsize_T *cts, win_T *wp, linenr_T lnum FUNC_ATTR
   cts->cts_line = line;
   cts->cts_ptr = ptr;
   cts->cts_cur_text_width = 0;
-  // TODO(bfredl): actually lookup inline virtual text here
```

Comment:
The doxygen comment above this function may also need change to stop mentioning "text properties", and `FUNC_ATTR_UNUSED` can be removed now.